app/src/main/res/drawable/splash_page.xml

```diff
@@ -1,30 +1,9 @@
-      android:fillColor="#fff"/>
-  <path
-      android:pathData="M270.5,264.87c0.24,-2.67 0.24,-5.2 -1.08,-7.53a8,8 0,0 0,-5.8 -3.94,17 17,0 0,0 -11,1.65 15.44,15.44 0,0 1,-1.42 0.67,1.43 1.43,0 0,1 -2,-0.57 9.1,9.1 0,0 1,-1.55 -3.44,2.26 2.26,0 0,1 0.68,-2.13 9.11,9.11 0,0 1,3.17 -2.12,18.5 18.5,0 0,1 6.44,-1.69 44.88,44.88 0,0 1,5.2 -0.17,15.67 15.67,0 0,1 10.12,4 14.14,14.14 0,0 1,4.52 8.61,25.57 25.57,0 0,1 -0.13,8.37c-0.2,1.31 -0.47,2.63 -0.71,4a1.09,1.09 0,0 0,0.07 0.57,57.81 57.81,0 0,1 2.36,8.3c0.27,1.49 0.44,3 0.61,4.56a2.18,2.18 0,0 1,0 0.67A1.49,1.49 0,0 1,278.4 286c-1.35,0 -2.67,0 -4,0a1.45,1.45 0,0 1,-1.62 -1.48c-0.2,-1.49 -0.44,-2.94 -0.64,-4.42a0.58,0.58 0,0 0,-0.07 -0.21c-0.51,0.71 -1,1.45 -1.55,2.13a13.42,13.42 0,0 1,-7.53 4.45,15.58 15.58,0 0,1 -10.76,-1.11 11.65,11.65 0,0 1,-6.95 -10.26,13 13,0 0,1 1.35,-7.12 11.35,11.35 0,0 1,6.48 -5.43,19.29 19.29,0 0,1 13.19,-0.07 14.31,14.31 0,0 1,3.91 2.23ZM260.69,268.11a12.55,12.55 0,0 0,-1.93 0.23,5.91 5.91,0 0,0 -4.52,3.78 5.44,5.44 0,0 0,2.94 6.85,8.64 8.64,0 0,0 4.32,0.64 9.23,9.23 0,0 0,6.47 -3.13,3.93 3.93,0 0,0 0.07,-5.54 6.82,6.82 0,0 0,-1.18 -1A11.33,11.33 0,0 0,260.69 268.11Z"
-      android:fillColor="#fff"/>
-  <path
-      android:pathData="M236.36,262.67v13.54a6.27,6.27 0,0 0,0.1 1.14,1.54 1.54,0 0,0 2.16,1.32c0.4,-0.14 0.81,-0.31 1.21,-0.47s0.58,-0.21 0.88,-0.31a1,1 0,0 1,1.38 0.58c0.54,1.08 1,2.22 1.49,3.34a2,2 0,0 1,-0.78 2.29,10.33 10.33,0 0,1 -5,2.23 11.81,11.81 0,0 1,-4.59 -0.17,5.86 5.86,0 0,1 -4.32,-3.88 15.61,15.61 0,0 1,-0.94 -5.57c0,-9.35 0,-18.69 0,-28a2.28,2.28 0,0 1,0.54 -1.62,1.76 1.76,0 0,1 1.41,-0.67h4.19a2.06,2.06 0,0 1,2.29 2.29v4.25Z"
-      android:fillColor="#fff"/>
-  <path
-      android:pathData="M232.28,228.33A5.52,5.52 0,0 1,238 234a5.67,5.67 0,0 1,-5.74 5.74,5.76 5.76,0 0,1 -5.73,-5.77v-0.07A5.65,5.65 0,0 1,232.28 228.33Z"
-      android:fillColor="#fff"/>
-</vector>
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+  <item android:drawable="@color/oppiaPrimaryLight" />
+  <item>
+    <bitmap
+      android:gravity="center"
+      android:src="@drawable/oppia_png_logo" />
```

Comment:
Could you upload webp versions of the images instead of png? webp are often smaller.

Reply:
Done